### ISSUE TITLE
E2E tests fails

### ISSUE THREAD

**Issue by maks-rafalko** [infection/infection#1901]
https://github.com/infection/infection/actions/runs/7120585613/job/19388267027#step:16:232

```
Box Requirements Checker
========================

> Using PHP 8.1.26
> PHP is using the following php.ini file:
  /etc/php/8.1/cli/php.ini

> Checking Box requirements:
  ......E

                                                                                
Error: ] Your system is not ready to run the application.                       
                                                                                

Fix the following mandatory requirements:
=========================================

 * The package "symfony/service-contracts" conflicts with the extension "psr".
   You need to disable it in order to run this application.

make[1]: Leaving directory '/home/runner/work/infection/infection'
make[1]: *** [Makefile:226: build/infection.phar] Error 1
make: *** [Makefile:57: compile] Error 2
Error: Process completed with exit code 2.
```

Extracted from https://github.com/infection/infection/pull/1900

**Comment by theofidry** [comment]
Hm I think it's an issue on Box side. Looks like `symfony/service-contracts` conflicts with `ext-psr: <1.1|>=2`. Back when I implemented the requirement checker checking the extensions versions was not a thing. It looks like it is now.

Meanwhile I guess it's best to disable the requirement checker.

**Comment by maks-rafalko** [comment]
> Meanwhile I guess it's best to disable the requirement checker.

could you advice how to do it so we can have a temp workaround?

**Comment by caendesilva** [comment]
Hey! I'm not familiar with this project, but just wanted to chime in and say that this was probably due to https://github.com/shivammathur/setup-php/issues/798, which was fixed in https://github.com/shivammathur/php-ubuntu/pull/3.

**Comment by sidz** [comment]
Seems like you are right @caendesilva. Thanks for reaching us out!

Fixed in https://github.com/infection/infection/pull/1906

**Comment by caendesilva** [comment]
> Seems like you are right @caendesilva. Thanks for letting know!
> 
> Fixed in #1906

No worries, glad I could help!
